apps/src/applab/appStorage.js

```diff
@@ -5,11 +5,17 @@
  */
 var AppStorage = module.exports;
 
-// TODO(dave): remove once we can store ids for each app.
-AppStorage.tempEncryptedAppId =
+// TODO(dave): remove once all applab data levels are associated with
+// a project.
+AppStorage.tempChannelId =
     window.location.hostname.split('.')[0] === 'localhost' ?
         "SmwVmYVl1V5UCCw1Ec6Dtw==" : "DvTw9X3pDcyDyil44S6qbw==";
 
+AppStorage.getChannelId = function() {
```

Comment:
Seems odd to use JS global variables to access data from the host (dashboard). We have traditionally used the appOptions passed during init to pass params to the app. Is that option available?